src/Microsoft.Windows.CsWin32/Generator.Com.cs

```diff
@@ -801,7 +803,7 @@ private unsafe (IReadOnlyList<MemberDeclarationSyntax> Members, IReadOnlyList<Ba
         if (populateVtblDeclared && this.IsFeatureAvailable(Feature.InterfaceStaticMembers) && !context.AllowMarshaling && GenerateCcwFor(originalIfaceName))
         {
             this.RequestComHelpers(context);
-            baseTypes.Add(SimpleBaseType(GenericName("IVTable").AddTypeArgumentListArguments(
+            baseTypes.Add(SimpleBaseType(GenericName($"{this.Win32NamespacePrefixString}.IVTable").AddTypeArgumentListArguments(
```

Comment:
Strictly speaking, I think this is an invalid `GenericName`. We should use a `ParseName` around this string before passing the result to `GenericName` so that we get a `QualifiedNameSyntax`.

Reply:
ParseName returns NameSyntax. I don't know how to turn that into a GenericName?

Reply:
All I know for sure is it's weird. And if this works, that's fine.